googleapiclient/http.py

```diff
@@ -546,14 +547,21 @@ def next_chunk(self, num_retries=0):
         }
     http = self._request.http
 
+    resp = None
```

Comment:
When SSLError happens, resp is not assigned and thus not defined which breaks line 556 which tries to use it in a condition.